### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,26 +6,26 @@
     <string name="calendar">Calendar</string>
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
-    <string name="apache"> Beta - Apache License 2.0</string>
+    <string name="apache"> Beta — Apache 2.0</string>
     <string name="kotlin">Kotlin</string>
     <string name="java">Java</string>
-    <string name="gnu">GNU General Public License CE</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml]: CE?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -6,26 +6,26 @@
     <string name="calendar">Calendar</string>
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
-    <string name="apache"> Beta - Apache License 2.0</string>
+    <string name="apache"> Beta — Apache 2.0</string>
     <string name="kotlin">Kotlin</string>
     <string name="java">Java</string>
-    <string name="gnu">GNU General Public License CE</string>
-    <string name="apachelicense">Apache License 2.0</string>
+    <string name="gnu">GPLv3+</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml]: Guessing "or any later version"

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -39,6 +39,6 @@
     <string name="followsys">  Follow System</string>
     <string name="light">  Light</string>
     <string name="dark">  Dark</string>
-    <string name="subItemTitle">Sub item title</string>
+    <string name="subItemTitle">Sub-item title</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml]: These are secondary titles of items?

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -39,6 +39,6 @@
     <string name="followsys">  Follow System</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:39]: Whitespace?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (CrazyMarvin) [reply app/src/main/res/values/strings.xml]: > CE?

Classpath Exception

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (comradekingu) [inline app/src/main/res/values/strings.xml]: Strange starting spaces for these.

**Comment by CrazyMarvin** [comment]
```
    <string name="followsys">  Follow System</string>
    <string name="light">  Light</string>
    <string name="dark">  Dark</string>
```

@arafaatqureshi, is it safe to remove the beginning whitespace? Is that needed for a workaround?

```
    <!-- TODO: Remove or change this placeholder text -->
    <string name="hello_blank_fragment">Hello blank fragment</string>
```

@arafaatqureshi, the "hello_blank_fragment" string can be removed, right?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: OK, but it doesn't say what version of the GPL it is.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="gnu">GPLv2+ with Classpath Exception</string>
```
?

Comment (arafaatqureshi) [reply app/src/main/res/values/strings.xml:12]: @CrazyMarvin Lemme see that why there are spaces before those strings, and yes it is safe to remove that string.

**Comment by arafaatqureshi** [comment]
> ```
>     <string name="followsys">  Follow System</string>
>     <string name="light">  Light</string>
>     <string name="dark">  Dark</string>
> ```
> 
> @arafaatqureshi, is it safe to remove the beginning whitespace? Is that needed for a workaround?
> 
> ```
>     
>     <string name="hello_blank_fragment">Hello blank fragment</string>
> ```
> 
> @arafaatqureshi, the "hello_blank_fragment" string can be removed, right?

@CrazyMarvin Lemme see that why there are spaces before those strings, and yes it is safe to remove that string.

**Comment by CrazyMarvin** [comment]
I removed the whitespaces with 19d42cf1619b72d867539d14031dec61426d383d.
@arafaatqureshi, may you please check if that has effects on the app?

**Comment by CrazyMarvin** [comment]
Hey @comradekingu,
I fixed the issues already in several commits in the meantime. Is it OK to close this issue or did I overlook something?
Thanks again for your support! 😘

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -4,26 +4,26 @@
     <string name="settings">Settings</string>
     <string name="timeline">Timeline</string>
     <string name="calendar">Calendar</string>
-    <string name="apache"> Beta - Apache License 2.0</string>
+    <string name="apache"> Beta — Apache 2.0</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:7]:
```suggestion
    <string name="apache">Beta — Apache 2.0</string>

```
?

**Comment by comradekingu** [comment]
@CrazyMarvin I fixed it up, should be good now.
I can be sure that at-least I won't be troubled by these not being fixed. :)

**Comment by CrazyMarvin** [comment]
I saw that you changed **Open-source licenses** into **Libre licenses**.
What is the difference between those terms? I don't know much about licenses but most programs I know which mention third-party open source licenses marked them as Open Source Licenses. 🤔

**Comment by comradekingu** [comment]
@CrazyMarvin
"Open source" specifically in English is an OSI trademarked term.
It was made to not focus on the freedoms, and doesn't communicate them well.
Much less in other languages, where it is not a trademarked term, making it quirky to say
~"source code that is open" without it sounding just like that or triggering further confusion.

Libre software is unencumbered by the FSF or OSI, and it does not get understood as freeware or gratis software.

**Comment by CrazyMarvin** [comment]
> @CrazyMarvin
> "Open source" specifically in English is an OSI trademarked term.
> It was made to not focus on the freedoms, and doesn't communicate them well.
> Much less in other languages, where it is not a trademarked term, making it quirky to say
> ~"source code that is open" without it sounding just like that or triggering further confusion.
> 
> Libre software is unencumbered by the FSF or OSI, and it does not get understood as freeware or gratis software.

Alright, then we will use libre licenses although it is almost the same for me but I see that it might differ in other languages.

**Pull request merged**